packages/cli/generators/interceptor/templates/interceptor-template.ts.ejs

```diff
@@ -19,9 +19,13 @@ import {
 <% if (isGlobal) { -%>
 @globalInterceptor('<%= group %>', {tags: {name: '<%= name %>'}})
 <% } else { -%>
-@bind({tags: {namespace: 'interceptors', name: '<%= name %>'}})
+@bind({tags: {key: <%= className %>.BINDING_KEY}})
 <% } -%>
 export class <%= className %> implements Provider<Interceptor> {
+<% if (!isGlobal) { -%>
+  static readonly BINDING_KEY = `interceptors.${<%= className %>.name}`;
```

Comment:
Discussion point: Should we emit `BINDING_KEY` property for global interceptors too? Such interceptors are typically picked up by the booter and applied automatically on all requests, therefore I feel there is no need to access the actual binding key.

Thoughts? /cc @strongloop/loopback-next